### PR TITLE
Replacing `tabwriter` with `pterm` and fixing `notify list` by adding usage

### PR DESCRIPTION
**Pull request by Ajpantuso**
### Summary
- Replacing `tabwriter` with `pterm` since `pterm` natively handles styling for headers.
- Fixing `ocm addons notify list` to recognize both `list` and the alias `ls`.

### CODE REVIEW

File: internal/output/table.go [hunk 1]
```diff
@@ -27,32 +25,24 @@ func NewTable(opts ...TableOption) (*Table, error) {
 			return nil, err
 		}
 
-		out = table.pager
+		table.cfg.Out = table.pager
 	}
 
-	table.writer = tabwriter.NewWriter(out, 0, tabWidth, 1, '\t', 0)
-
 	if !table.cfg.NoHeaders {
```
Comment (sandhyadalavi) [inline internal/output/table.go:31]: do we need this condition?

Comment (sandhyadalavi) [reply internal/output/table.go:31]: other than this, LGTM

Comment (Ajpantuso) [reply internal/output/table.go:31]: Yes. `pterm` does not actually track headers separate from the table data itself so we need to ensure the headers are not written ourselves.

**Review by sandhyadalavi — APPROVED** [review]
lgtm

**Pull request merged**